pkg/cmd/run_node.go

```diff
@@ -33,6 +33,11 @@ func ParseConfig(cmd *cobra.Command) (rollconf.Config, error) {
 		return rollconf.Config{}, fmt.Errorf("failed to load node config: %w", err)
 	}
 
+	if nodeConfig.Node.Pruning.Strategy != rollconf.PruningConfigStrategyCustom {
```

Comment:
Should this be default instead?
```suggestion
	if nodeConfig.Node.Pruning.Strategy != rollconf.PruningConfigStrategyDefault {
```

Reply:
`custom` is configured by user. Other strategies need to ensure using corresponding strategy